src/Streamly/Internal/Data/Parser/ParserD.hs

```diff
@@ -377,7 +349,7 @@ takeGE cnt (Fold fstep finitial fextract) = Parser step initial extract
                       FL.Done _ ->
                         Error
                             $ "takeGE: the collecting fold terminated after "
-                                ++ "consuming " ++ show i1 ++ " elements"
+                                ++ "consuming" ++ show i1 ++ " elements"
```

Comment:
Why remove the space?

Reply:
That's by mistake, some issue during transferring the changes from one branch to another.